ru/turing/courses/lesson2/animals/bird.java

```diff
@@ -0,0 +1,18 @@
+package ru.turing.courses.lesson2.animals;
+
+public class bird extends Animal {
+    private boolean migration;
+    //true - letit na south,false-no
+    private String singing;
+
+    public bird(String vid, boolean eatingmeat, int age, boolean wild, boolean migration, String singing) {
+        super(vid, eatingmeat, age, wild);
+        this.migration = migration;
+        this.singing = singing;
+    }
+
+    public void makeSound() {
```

Comment:
```suggestion
    @Override
    public void makeSound() {
```

Здесь очень не помешал бы оверрайд.